frontend/src/utils/templates.ts

```diff
@@ -58,3 +58,13 @@ const replacePlaceholders = (baseText: string, replacementConfig: { [key: string
   }
   return newText;
 };
+
+export const confirmationTemplate = (replaceConfig: { [key: string]: string | string[] | undefined }) => {
+  const baseTemplate =
+    'Dear {{ contactName }},\n\n' +
+    'The confirmation ID for your submission is: {{ activityId }}.\n\n' +
+    'A Housing Navigator will review your submission and contact you. Keep the Confirmation ID for future reference.\n\n' +
```

Comment:
Getting some line length linter warnings here. As this probably can't be broken down due to templating we should add `// eslint-disable-next-line max-len`